p4src/linux_networking/README_LINUX_NETWORKING_WITH_ECMP.md

```diff
@@ -216,17 +238,55 @@ Note: Port number used in p4rt-ctl commands are target datapath indexes(unique i
     p4rt-ctl add-entry br0 linux_networking_control.handle_tx_control_pkts_table "istd.input_port=5,action=linux_networking_control.set_control_dest(4)"
   ```
 
-#### 14. Configure ECMP for underlay connectivity
+#### 15. Configure ECMP for underlay connectivity
 - **Rule for: To reach link partner multiple paths are configured, packets can be hashed to any port where the nexthop's ARP is learnt**
 - **Nexthop is selected based on 5 tuple parameter. Which are source IPv4 address, destination IPv4 address, protocol type, UDP source port and UDP destination port of the overlay packet**
```

Comment:
"Nexthop is selected based on a 5-tuple parameter: source IPv4 address, destination IPv4 address, ..."

Is this "a 5-tuple parameter" or is it "five parameters?"

Reply:
General terminology is 5-tuple parameter. Can see similar 5-tuple params used in RFC 6146.